packages/lambda/src/client.ts

```diff
@@ -26,6 +26,7 @@ import type {AwsRegion} from './pricing/aws-regions';
 import type {RenderProgress} from './shared/constants';
 import type {WebhookPayload} from './shared/invoke-webhook';
 
+export {CustomCredentials, getServiceClient} from './shared/aws-clients';
```

Comment:
This export `getServiceClient` instead of `getAwsClient` 

Consequently `import { getAwsClient } from "@remotion/lambda/client";` doesn't work
@JonnyBurger